qcfractal/interface/client.py

```diff
@@ -387,12 +388,13 @@ def add_compute(self, program, method, basis, driver, options, molecule_id, retu
                 "program": program,
                 "method": method,
                 "basis": basis,
-                "options": options
+                "options": options,
+                "tag": tag,
             },
             "data": molecule_id
         }
 
-        r = self._request("post", "task_scheduler", payload)
```

Comment:
I think task_queue was more appropriate.

Reply:
We have two different APIs:
1) where a users push/pulls new task or examine tasks already pushed in
2) where a manager pulls tasks to run and pushes back completed tasks.

I think this should probably be two different APIs (especially as permissions are different). But they are also "paired" APIs. If this is `task_queue`, what should the manager API be called?

Reply:
I think the task submission (post/get) is a separate API as you have it. It's a user API where they submit tasks and query their status. As a RESTful API, the endpoint /task is common.

For the queue manager, it's a different API as you pointed. It consumes tasks from the queue, so make sense to call it queue_manager, or just queue (post/get).

Reply:
Looking at the question again, I moved this from `task_scheduler` to `task_queue. Did you mean I should move it back to `task_scheduler` or keep it as `task_queue`?

Reply:
I agree, they are two APIs. One submits tasks and query about them, say /task POST and /task GET; and the other is related to the queue, e.g., /queue POST and /queue GET.